admin_guide/scheduling/scheduler.adoc

```diff
@@ -0,0 +1,827 @@
+target particular nodes. The constraint on the use of the `nodeSelector` field
+of a pod configuration is based on the *pods/binding* permission and the
+`nodeSelectorLabelBlacklist` configuration option.
+====
+
+The `nodeSelectorLabelBlacklist` field of a master configuration file gives you
+control over the labels that certain roles can specify in a pod configuration's
+`nodeSelector` field. Users, service accounts, and groups that have the
+*pods/binding* permission can specify any node selector. Those without the
+*pods/binding* permission are prohibited from setting a `nodeSelector` for any
+label that appears in `nodeSelectorLabelBlacklist`.
+
+As a hypothetical example, an {product-title} cluster might consist of five data
+centers spread across two regions. In the U.S., "us-east", "us-central", and
+"us-west"; and in the Asia-Pacific region (APAC), "apac-east" and "apac-west".
+Each node in each geographical region is labeled accordingly. For example,
+`region: us-east`.
+ifdef::openshift-enterprise,openshift-origin[]
+[NOTE]
+====
+See xref:../../admin_guide/manage_nodes.adoc#updating-labels-on-nodes[Updating
+Labels on Nodes] for details on assigning labels.
+====
+endif::openshift-enterprise,openshift-origin[]
+
+As a cluster administrator, you can create an infrastructure where application
+developers should be deploying pods only onto the nodes closest to their
+geographical location. You can
+xref:../../admin_guide/managing_projects.adoc#using-node-selectors[create a node
+selector], grouping the U.S. data centers into `superregion: us` and the APAC
+data centers into `superregion: apac`.
+
+To maintain an even loading of resources per data center, you can add the
+desired `region` to the `nodeSelectorLabelBlacklist` section of a master
+configuration. Then, whenever a developer located in the U.S. creates a pod, it
+is deployed onto a node in one of the regions with the `superregion: us` label.
+If the developer tries to target a specific region for their pod (for example,
+`region: us-east`), they receive an error. If they try again, without the
+node selector on their pod, it can still be deployed onto the region they tried
+to target, because `superregion: us` is set as the project-level node selector,
+and nodes labeled `region: us-east` are also labeled `superregion: us`.
+
+[[constraining-pod-placement-labels]]
+=== Constraining Pod Placement Using Node Name
+
+Ensure a pod is deployed onto only a specified node host by assigning it a label
+and specifying this in the `nodeName` setting in a pod configuration.
+
+. Ensure you have the desired labels and
+xref:../../admin_guide/managing_projects.adoc#using-node-selectors[node selector]
+set up in your environment.
++
+For example, make sure that your pod configuration features the `nodeName`
+value indicating the desired label:
++
+----
+apiVersion: v1
+kind: Pod
+spec:
+  nodeName: <key: value>
```

Comment:
@aveshagarwal @wmengRH 

Is that a literal string? 'nodeName: hostxxxx.redhat.com`